sessionctx/variable/session.go

```diff
@@ -1330,6 +1337,11 @@ func (s *SessionVars) SetSystemVar(name string, val string) error {
 		config.GetGlobalConfig().CheckMb4ValueInUTF8 = TiDBOptOn(val)
 	case TiDBFoundInPlanCache:
 		s.FoundInPlanCache = TiDBOptOn(val)
+	case TiDBFoundInSPM:
+		if val == "0" {
+			panic("1111")
+		}
```

Comment:
Why panic here?

Reply:
Sorry. This is for debugging. I will delete it.